### DISCUSSION TITLE
Is it possible to train ChatGPT-Writer ?

### DISCUSSION THREAD

**Discussion by CamCall1000** [GorvGoyl/ChatGPT-Writer#77] [Provide Feedback and Ideas]
Hi, 

Thanks so much for you work and this amazing product ! 

I am using it to answer quicker to client enquiries (about product availability, project faisability, price, etc.) It's quite redundant enquiry but still need a bit of personalization. Like a junior sales could answer to it with little bit of knowledge about the business. 

I am wondering, is it possible to train ChatGPT-Writer on data about my company, my sales catalog and stocks to provide detailed answer almost automatically ?  

Best,
Cam

**Comment by wiiiktor** [comment]
Yeah, using fine-tuned ChatGPT with my own API would offer this functionality - is this planned?..
